massa-models/src/address.rs

```diff
@@ -158,6 +160,30 @@ impl Address {
     }
 }
 
+/// Deserializer for `Address`
+#[derive(Default)]
+pub struct AddressDeserializer {
+    hash_deserializer: HashDeserializer,
+}
+
+impl AddressDeserializer {
+    /// Creates a new deserializer for `Address`
+    pub fn new() -> Self {
+        Self {
+            hash_deserializer: HashDeserializer::new(),
+        }
+    }
+}
+
+impl Deserializer<Address> for AddressDeserializer {
+    fn deserialize<'a>(&self, buffer: &'a [u8]) -> IResult<&'a [u8], Address> {
+        Ok((
+            &buffer[ADDRESS_SIZE_BYTES..],
```

Comment:
this might panic if buffer is smaller than ADDRESS_SIZE_BYTES

Reply:
Refactored.